src/pages/library/Datasets.js

```diff
@@ -118,7 +130,8 @@ const thousandGenomesHighCoverage = () => h(Participant, {
   h(ButtonPrimary, {
     'aria-label': 'Browse 1000 Genomes High Coverage data',
     tooltip: 'Visit the workspace',
-    href: Nav.getLink('workspace-dashboard', { namespace: 'anvil-datastorage', name: '1000G-high-coverage-2019' })
+    href: Nav.getLink('workspace-dashboard', { namespace: 'anvil-datastorage', name: '1000G-high-coverage-2019' }),
+    onClick: () => captureBrowseDataEvent('1000G-high-coverage-2019')
```

Comment:
Would it be possible to pass in the current object and then use the `aria-label` property? (Or maybe some other way to do it that doesn't involve duplication?)

Reply:
I could pass it in here, but I'm not sure if that would make it more readable: https://github.com/DataBiosphere/terra-ui/blob/86af332a65d3e27e9955288e9b72a619ae13cfbd/src/pages/library/Datasets.js#L442

Reply:
What do you think about passing in just the aria-label, like thousandGenomesHighCoverage('Browse` 1000 Genomes High Coverage data')?